src/System.Globalization/tests/CompareInfo/CompareInfoCompare.cs

```diff
@@ -561,6 +513,18 @@ public void Test148()
         [Fact]
         public void Test156() { Test(CultureInfo.InvariantCulture, new string('a', 5555), new string('a', 5555), 0, CompareOptions.None); }
 
+        [Fact]
+        public void Test157() { Test(CultureInfo.InvariantCulture, "foobar", "FooB\u00C0R", 0, CompareOptions.IgnoreNonSpace | CompareOptions.IgnoreCase); }
+
+        [Fact]
+        public void Test158() { Test(CultureInfo.InvariantCulture, "foobar", "FooB\u00C0R", -1, CompareOptions.IgnoreNonSpace); }
+
+        [Fact]
+        public void Test159() { Test(CultureInfo.InvariantCulture, "\uFF66", "\u30F2", 0, CompareOptions.IgnoreWidth); }
+
+        [Fact]
+        public void Test160() { Test(CultureInfo.InvariantCulture, "\uFF66", "\u30F2", 1, CompareOptions.IgnoreCase); }
```

Comment:
This test appears to be failing on Windows:
```
System.Globalization.Tests.CompareInfoCompare.Test160 [FAIL]
        Assert.Equal() Failure
        Expected: 1
        Actual:   -1
        Stack Trace:
           d:\j\workspace\dotnet_corefx_windows_release_prtest\src\System.Globalization\tests\CompareInfo\CompareInfoCompare.cs(534,0): at System.Globalization.Tests.CompareInfoCompare.Test(CultureInfo culture, String str1, String str2, Int32 expected, CompareOptions options)
           d:\j\workspace\dotnet_corefx_windows_release_prtest\src\System.Globalization\tests\CompareInfo\CompareInfoCompare.cs(526,0): at System.Globalization.Tests.CompareInfoCompare.Test160()
```
This shouldn't depend on your coreclr changes, right?